x-pack/plugins/fleet/server/services/epm/packages/bulk_install_packages.ts

```diff
@@ -55,10 +61,32 @@ export async function bulkInstallPackages({
     return pkg.name;
   });
 
-  const limiter = pLimit(10);
+  const forbiddenPackages = uniquePackages
+    .filter((pkg) =>
+      typeof pkg === 'string'
+        ? FORBIDDEN_BULK_INSTALL_PACKAGE_NAMES.includes(pkg)
+        : FORBIDDEN_BULK_INSTALL_PACKAGE_NAMES.includes(pkg.name)
+    )
+    .map((pkg) => (typeof pkg === 'string' ? pkg : pkg.name));
+
+  if (forbiddenPackages.length > 0) {
+    logger.warn(
+      `Package(s) cannot be bulk installed and will be ignored: ${forbiddenPackages.join(',')}`
+    );
+  }
```

Comment:
Realizing now that a better thing to do would be to include this as a `not_installed` status in the response.